src/Processors/Merges/Algorithms/SummingSortedAlgorithm.cpp

```diff
@@ -527,6 +545,7 @@ void SummingSortedAlgorithm::SummingMergedData::initialize(const DB::Block & hea
     columns = std::move(new_columns);
 
     current_row.resize(def.column_names.size());
+    current_row_columns.resize(def.column_names.size());
```

Comment:
nitpick: even though we always call `initialize` before any other method, initializing vector of proper size if ctor looks more robust, since we already have `def` there